tests/all_test.py

```diff
@@ -1538,3 +1538,27 @@ def setUp(self):
 
     def test_parsing(self):
         _params_tests(self)
+
+@unittest.skipUnless(LIVE_TEST, "requires HOST variable to be set")
+class TestRawBytes(unittest.TestCase):
```

Comment:
I think you should inherit DbTestCase class instead of unittest.TestCase here, this should fix the test